content/docs/addons-shallow-renderer.md

```diff
@@ -47,22 +47,22 @@ expect(result.props.children).toEqual([
 ]);
 ```
 
-Shallow testing currently has some limitations, namely not supporting refs.
+वर्तमान में शैलो टेस्टिंग की कुछ सीमाएँ हैं, नामतः ये refs के साथ काम नहीं करता।
 
-> Note:
+> टिप्पणी:
 >
-> We also recommend checking out Enzyme's [Shallow Rendering API](https://airbnb.io/enzyme/docs/api/shallow.html). It provides a nicer higher-level API over the same functionality.
+> हम Enzyme के [शैलो रेंडरिंग API](https://airbnb.io/enzyme/docs/api/shallow.html) का भी परीक्षण करने की भी सलाह देते हैं। यह समान कार्यक्षमता पर एक अच्छा उच्च स्तरीय API प्रदान करता है।
```

Comment:
यह समान कार्यक्षमता पर एक अच्छा उच्च स्तरीय API प्रदान करता है। -> can be simplified a bit?

Reply:
`यह उसी फंक्शनलिटी पर एक और अच्छे स्तर की API देता है` ??

Reply:
sounds good